subsys/dfu/fmfu_fdev/include/modem_update_decode.h

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
Thsse autogenerated files do not comply with NCS coding style. In general, license is missing, pointers have `p_` prefix and C++ comment is used in the end.  Not sure though if it's required to follow the rules for such headers/sources. CC @carlescufi

Reply:
> Not sure though if it's required to follow the rules for such headers/sources.

If things are committed into the repository, they should follow the coding guidelines, even when generated.
A user checking out the repository will look at the code and not consider whether it was written by hand or generated.

Reply:
I have added licence text and clang-format. We will update the generator to avoid `p_` prefixes and c++ comments.